osCam/core/urls.py

```diff
@@ -6,7 +6,8 @@
 # global TYPE="stream"
 
 urlpatterns = [
-    path('', views.home, name='home'),    
+    path('', views.home, name='home'),   
+    # path('<str>/',views.getFilePath, name='filepath') ,
     path('feed/', views.feed, name='feed'),
     # path('/settings/', views.settings, name='settings'),
```

Comment:
should the path to settings be commented out? did you move this path somewhere else? If so, remove the comment.

Reply:
The one on the right hand side is the code changes I've made and LHS are code changes that were already part of Upstream. I guess settings page was not being used here because I just checked and it still works with all the forms that were made in settings page